invenio_vocabularies/contrib/names/mappings/os-v1/names/name-v2.0.0.json

```diff
@@ -125,6 +125,17 @@
             "type": "text",
             "analyzer": "accent_edge_analyzer",
             "search_analyzer": "accent_analyzer"
+          },
+          "acronym": {
+            "type": "text",
+            "analyzer": "accent_edge_analyzer",
```

Comment:
Comment: Checking the mappings seems that `identifiers` which are included in the fields to be searched, do not have the same analyzer as acronym and name, therefore I don't think that searching for Acronym and Identifier name will work. However, this might not be an important use case to cover. Up to you.

Reply:
`identifiers.identifier` is there so that one can search explicitly e.g. by ORCID (or other name identifiers), basically copy-paste what they're searching for in the searchbar